src/main/java/com/manywho/services/dummy/identity/IdentityController.java

```diff
@@ -85,6 +90,22 @@ public ObjectDataResponse authorization(ObjectDataRequest objectDataRequest) thr
         $User userObject;
         URI host = baseUri(httpHeaders.getHeaderString("X-Forwarded-Proto"));
 
+        String token = httpHeaders.getHeaderString("Authorization");
+        String decodedToken = URLDecoder.decode(token, "UTF-8");;
+
+        Map<Object, Object> parameters = Splitter.on(
+                "&")
+                .withKeyValueSeparator("=")
+                .split(decodedToken)
+                .entrySet()
+                .stream()
+                .collect(Collectors.toMap(
+                        entry -> CaseFormat.UPPER_CAMEL.to(CaseFormat.LOWER_CAMEL, entry.getKey()),
+                        entry -> entry.getValue()
+                ));
+
+        authenticatedWho = new ObjectMapper().convertValue(parameters, AuthenticatedWho.class);
```

Comment:
Hmm... I'm a bit puzzled - 'Authorization' header is not containing a JWT token?

Reply:
Yeah, this block was added in to work around a bug in the `authenticatedWhoProvider` which was subsequently fixed but I had forgotten to remove this workaround. Removed the workaround.